.github/workflows/example.yml

```diff
@@ -89,16 +89,22 @@ jobs:
   # Example of building coreboot
   # ANCHOR: example_build_coreboot
   build-coreboot:
-    runs-on: ubuntu-latest
```

Comment:
Could you briefly explain why this got removed? Otherwise the PR looks fine to me.

Reply:
Yes, it is replaced with
```yaml
runs-on: ${{ matrix.arch == 'arm64' && 'ARM64' || 'ubuntu-latest' }}
```

So basically it picks whether to run on `ubuntu-latest` or our self-hosted runner.

Reply:
Oh I missed that line, thanks for the explanation :+1: